theme_vehicle/views/customizations.xml

```diff
@@ -82,19 +82,19 @@
     <xpath expr="//section" position="attributes">
         <attribute name="data-masonry-template">image_text_image</attribute>
     </xpath>
+    <xpath expr="//section/div" position="attributes">
+        <attribute name="class" add="container-fluid" remove="container" separator=" "></attribute>
+    </xpath>
 </template>
 
 <template id="s_masonry_block_image_texts_image_template" inherit_id="website.s_masonry_block_image_texts_image_template" name="Vehicle s_masonry_block">
-    <xpath expr="//div[hasclass('col-lg-3')]" position="replace" mode="inner">
-        <img src="/web/image/website.s_parallax_default_image" class="img img-fluid mx-auto" alt=""/>
-    </xpath>
```

Comment:
With the parallax image, it was looking like that before 
![image](https://github.com/user-attachments/assets/12b75b63-18ff-42a9-a463-fd4669808de3)

But now, since there is no second masonry block image, it defaults to the default one and looks like this
![image](https://github.com/user-attachments/assets/dec18c1b-cc09-4fbf-b7b1-34a37c8c945b)

Maybe it is worth keeping the parallax image ? :slightly_smiling_face:

Reply:
Ooops, thank you, fixed 👍